docs/quickstart.rst

```diff
@@ -129,6 +129,28 @@ certain number of old archives::
     borg prune -v $REPOSITORY --prefix '{hostname}-' \
         --keep-daily=7 --keep-weekly=4 --keep-monthly=6
 
+Pitfalls with shell variables and environment variables
+-------------------------------------------------------
+
+This applies to all environment variables you want borg to see, not just
+``BORG_PASSPHRASE``. The short explanation is: always ``export`` your variable,
+and use single quotes if you're unsure of the details of your shell's expansion
+behavior. E.g.::
+
+    export BORG_PASSPHRASE='complicated & long';
+
+This is because ``export`` exposes variables to subprocesses, which borg may be
+one of. More on ``export`` can be found in the "ENVIRONMENT" section of the
+bash(1) man page.
+
+.. note:: ``sudo`` does not inherit your environment, so ``sudo borg ...`` will not
+          have access to ``BORG_PASSPHRASE`` despite running the above example,
+          but ``sudo export BORG_PASSPHRASE='complicated & long' borg ...``
```

Comment:
This doesn‘t seem to work. without the export it seems to work. But sudo is complicated, and calling like this will leave BORG_PASSPHRASE visible to ps in some configurations. sudo is really quite complicated.

Maybe we should rather go

> For options to pass environment variables through sudo consider using the ``env_keep`` option in suders or the ``sudo BORG_PASSPHRASE='complicated & long' borg`` syntax. Values passed as parameters to sudo might be visible to other users.